pkg/templateservicebroker/brokerapi/templateservicebroker/provision.go

```diff
@@ -0,0 +1,186 @@
+package templateservicebroker
+
+import (
+	"net/http"
+	"reflect"
+
+	templateapi "github.com/openshift/origin/pkg/template/api"
+	"github.com/openshift/origin/pkg/templateservicebroker"
+	"github.com/openshift/origin/pkg/templateservicebroker/api"
+	"github.com/openshift/origin/pkg/templateservicebroker/brokerapi"
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/errors"
+	"k8s.io/kubernetes/pkg/types"
+)
+
+func (b *Broker) templateFromUID(uid types.UID) (*templateapi.Template, error) {
+	// TODO: cache this?
```

Comment:
no, the informer is kept up to date by the inner workings of the informer implementation (it sets up a watcher on etcd and keeps itself up to date).  The controller gets notified *by* the informer (if there is a controller that cares).